assets/src/edit-story/icons/index.js

```diff
@@ -14,77 +14,11 @@
  * limitations under the License.
  */
 
-export { ReactComponent as Add } from './add.svg';
-export { ReactComponent as AddColor } from './color_add.svg';
-export { ReactComponent as AlignBottom } from './align_bottom.svg';
-export { ReactComponent as AlignCenter } from './align_center.svg';
-export { ReactComponent as AlignCenterAlt } from './center_align.svg';
-export { ReactComponent as AlignLeft } from './align_left.svg';
-export { ReactComponent as AlignLeftAlt } from './left_align.svg';
-export { ReactComponent as AlignMiddle } from './align_middle.svg';
-export { ReactComponent as AlignMiddleAlt } from './middle_align.svg';
-export { ReactComponent as AlignRight } from './align_right.svg';
-export { ReactComponent as AlignRightAlt } from './right_align.svg';
-export { ReactComponent as AlignTop } from './align_top.svg';
-export { ReactComponent as Arrow } from './arrow.svg';
-export { ReactComponent as ArrowLeft } from './arrow_left.svg';
-export { ReactComponent as ArrowRight } from './arrow_right.svg';
 export { ReactComponent as ArrowDown } from './arrow_down.svg';
```

Comment:
We should probably create tickets to update all the components using these to using the new design system icons (and making sure they exist there and in Figma).